community-operators/buildv2-operator/0.1.0/buildv2-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,251 @@
+    capabilities: Basic Install
+    categories: Developer Tools, OpenShift Optional
+    containerImage: quay.io/shipwright/shipwright-operator:v0.1.0
+    createdAt: "2020-10-03T17:44:55Z"
+    description: Build lean container images of your application on Kubernetes using
+      buildah, s2i, buildpacks, kaniko, and more
+    repository: https://github.com/shipwright-io/build
+  name: buildv2-operator.v0.1.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: BuildRun is the Schema representing an instance of build execution
+      kind: BuildRun
+      name: buildruns.build.dev
+      version: v1alpha1
+      statusDescriptors:
+        - description: Indicates if the build execution was successful.
+          displayName: Succeeded
+          path: succeeded
+          x-descriptors:
+          - urn:alm:descriptor:text
+        - description: Indicates the reason for build execution failure.
+          displayName: Reason
+          path: reason
+          x-descriptors:
+          - urn:alm:descriptor:text
+    - description: Build is the Schema representing a Build definition
+      kind: Build
+      name: builds.build.dev
+      version: v1alpha1
+      statusDescriptors:
+        - description: Indicates if the build registration was successful.
+          displayName: Reason
+          path: reason
+          x-descriptors:
+          - urn:alm:descriptor:text
+        - description: Indicates the reason for Build Registration failure
+          displayName: Registered
+          path: registered
+          x-descriptors:
+          - urn:alm:descriptor:text
+    - description: BuildStrategy is the Schema representing a strategy in the namespace
+        scope to build images from source code.
+      kind: BuildStrategy
+      name: buildstrategies.build.dev
+      version: v1alpha1
+    - description: ClusterBuildStrategy is the Schema representing a strategy in the
+        cluster scope to build images from source code.
+      kind: ClusterBuildStrategy
+      name: clusterbuildstrategies.build.dev
+      version: v1alpha1
+    required:
+    - description: Represents Installation of Tekton.
+      displayName: OpenShift Pipelines Config
+      kind: Config
+      name: config.operator.tekton.dev
```

Comment:
@J0zi 
This is a productized operator not present in community-operators. "Build v2" operator depends on a CRDs delivered from this "required" operator. The dependency resolution works well on OpenShift, is it possible that the resolution is failing in the CI job here? Any way I could get logs of the operator startup failure?